lib/gollum/public/gollum/stylesheets/app.scss

```diff
@@ -1,8 +1,102 @@
-//= require primer
+//= require template
 //= require editor
-//= require dialog
 //= require criticmarkup
 //= require tables
 //= require emoji
-//= require template
-//= require spinner
\ No newline at end of file
+//= require dialog
+//= require highlights
+
+//  Simply importing @primer/css/index.scss also imports the unneeded marketing styles, so import the following components manually:
+//= require @primer/css/color-modes
+//= require @primer/css/core
+//= require @primer/css/product
+
+@import '_base';
+
+body {
+    font: 13.34px $font-default;
```

Comment:
@bartkamphorst @benjaminwil: I've moved this here from `template.scss` in order to reproduce the current styling as it exists on `master`. However, we could also leave this out and go with the (new) primer defaults. The latter also looks nice to me; I have no strong preference. Thoughts?

Reply:
No real preference, although I think using the Primer defaults as much as possible benefits all of us.